bot/exts/utils/_eval_helper.py

```diff
@@ -0,0 +1,339 @@
+                        f"status code: {res.status}"
+                    )
+
+                data = await res.read()
+                data = data.decode("utf-8")
+                return data.replace(data[:16], "")  # remove token
+
+
+class EvalHelper:
+    """Eval Helper class."""
+
+    def __init__(self, language: str):
+        self.lang = language.strip("`").lower()
+        self.authorized = (
+            "https://hastebin.com",
+            "https://gist.github.com",
+            "https://gist.githubusercontent.com",
+        )
+        self.max_file_size = 20000
+        self.truncated_error = "The output exceeded 128 KiB and was truncated."
+        self.hastebin_link = "https://hastebin.com"
+        self.bin_link = "https://bin.drlazor.be/"
+
+    async def parse(
+        self, code: str
+    ) -> Tuple[
+        str, str, str, Dict[Union[str, Any], bool], List[str], List[str], List[str]
+    ]:
+        """Returned parsed data."""
+        options = {"--stats": False, "--wrapped": False}
+        options_amount = len(options)
+
+        # Setting options and removing them from the beginning of the command
+        # options may be separated by any single whitespace, which we keep in the list
+        code = re.split(r"(\s)", code, maxsplit=options_amount)
+        for option in options:
+            if option in code[: options_amount * 2]:
+                options[option] = True
+                i = code.index(option)
+                code.pop(i)
+                code.pop(i)  # Remove following whitespace character
+        code = "".join(code)
+
+        compiler_flags = []
+        command_line_options = []
+        args = []
+        inputs = []
+
+        lines = code.split("\n")
+        code = []
+        for line in lines:
+            if line.startswith("input "):
+                inputs.append(" ".join(line.split(" ")[1:]).strip("`"))
+            elif line.startswith("compiler-flags "):
+                compiler_flags.extend(line[15:].strip("`").split(" "))
+            elif line.startswith("command-line-options "):
+                command_line_options.extend(line[21:].strip("`").split(" "))
+            elif line.startswith("arguments "):
+                args.extend(line[10:].strip("`").split(" "))
+            else:
+                code.append(line)
+
+        inputs = "\n".join(inputs)
+        code = "\n".join(code)
+        return (
+            inputs,
+            code,
+            self.lang,
+            options,
+            compiler_flags,
+            command_line_options,
+            args,
+        )
+
+    async def code_from_attachments(self, ctx: Context) -> Optional[str]:
+        """Code in file."""
+        file = ctx.message.attachments[0]
+        if file.size > self.max_file_size:
+            await ctx.send("File must be smaller than 20 kio.")
+            logger.info("Exiting | File bigger than 20 kio.")
+            return None
```

Comment:
`return` returns `None` by default.
```suggestion
            return
```